apps/src/blockly/eventHandlers.ts

```diff
@@ -39,6 +39,7 @@ export function disableOrphans(event: GoogleBlockly.Events.Abstract) {
   if (
     event.type !== Blockly.Events.BLOCK_CHANGE &&
     event.type !== Blockly.Events.BLOCK_MOVE &&
+    event.type !== Blockly.Events.BLOCK_DRAG &&
```

Comment:
Nit: With so many potential types to check here, I wonder if it would be easier to refactor them into a list and use `.includes(event.type)`.